src/backend/distributed/executor/multi_utility.c

```diff
@@ -257,20 +253,17 @@ multi_ProcessUtility(Node *parsetree,
 	 */
 	if (EnableDDLPropagation)
 	{
-		bool isTopLevel = (context == PROCESS_UTILITY_TOPLEVEL);
-
 		if (IsA(parsetree, IndexStmt))
 		{
-			parsetree = ProcessIndexStmt((IndexStmt *) parsetree, queryString,
-										 isTopLevel);
+			ddlJob = PlanIndexStmt((IndexStmt *) parsetree, queryString);
```

Comment:
Hm, I was kind of hoping we'd now be in a position to support things like `CREATE INDEX ON ..`  without an explicit name. What if we only keep `DropStmt` above `standard_ProcessUtility`?